ouroboros-consensus-mock/src/Ouroboros/Consensus/Mock/Ledger/Block.hs

```diff
@@ -80,9 +80,12 @@ import           Cardano.Binary (ToCBOR (..))
 import           Cardano.Crypto.Hash (Hash, HashAlgorithm, MD5, ShortHash)
 import qualified Cardano.Crypto.Hash as Hash
 
+import           Ouroboros.Network.Block
```

Comment:
I generally don't expect to see this import added anywhere. Do you know what exactly its bringing in scope here? I anticipate there is a `Consensus` module that re-exports that thing, whatever it is.

Reply:
It brings `Serialised` (defined directly in `Ouroboros.Network.Block`)  into scope which is used [below](https://github.com/input-output-hk/ouroboros-network/pull/3214/files#diff-b3aa9feded0c37a4ee5a60931752113f71b8f12a2a109c6645198b9cca26c9ceR142) for the `SerialiseNodeToClientConstraints (SimpleBlock c ext)` instance. So this is not something that can be trivially changed.

Reply:
Yep, looks like no `Consensus` module re-exports that one :(

Would you write an explicit import list on that one? It will avoid me wondering this same question at a future date :)